drivers/ra/CMakeLists.txt

```diff
@@ -57,6 +57,8 @@ zephyr_library_sources_ifdef(CONFIG_USE_RA_FSP_SDHI
 	fsp/src/r_sdhi/r_sdhi.c)
 zephyr_library_sources_ifdef(CONFIG_USE_RA_FSP_OSPI_B_NOR_FLASH
 	fsp/src/r_ospi_b/r_ospi_b.c)
+zephyr_library_sources_ifdef(CONFIG_USE_RA_FSP_SDRAM
+	fsp/src/bsp/mcu/all/bsp_sdram.c)
```

Comment:
How does this SDRAM controller require in display driver support ?

Reply:
Because the MIPI display in the EK-RA8D1 kit has large resolution, so the display buffer is too large to place in internal RAM of RA8D1 MCU. So, it required the external RAM (SDRAM) support to work with GUI app.
This SDRAM controller not only support the RAM part on EK-RA8D1 board but also support other SDRAM on other boards by setting the input config in devicetree. This is 1 of request from reviewer, that the SDRAM controller in Zephyr should be a generic driver to work with any SDRAM part.